client/components/tinymce/plugins/contact-form/dialog/field-edit-button.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import Popover from 'components/popover';
+
+export default React.createClass( {
+	displayName: 'ContactFormDialogFieldEditButton',
+
+	mixins: [ PureRenderMixin ],
+
+	propTypes: {
+		expanded: PropTypes.bool.isRequired
+	},
```

Comment:
For a stateless function we can add directly:
```javascript
ContactFormDialogFieldEditButton.propTypes = {

};
```

For a class this can be defined as a [static](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static) property:
```javascript
static propTypes = {
    expanded: PropTypes.bool.isRequired
}
```